modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/OperationReply.java

```diff
@@ -579,11 +579,15 @@ public SequenceTag buildSequence(Operation operation, SequenceTag sequenceElemen
                 if (policy.getCode().equals(PluginConstants.OperationCodes.PASSCODE_POLICY)) {
```

Comment:
Possible NPE as CONSTANT is checked at last.
Should be PASSCODE_POLICY.equals(policy.getCode()).